Applications/Resnet/README.md

```diff
@@ -0,0 +1,30 @@
+# Resnet with cifar100
+
+This application contains a Resnet18 model and a trainer with cifar100.
+
+Reference. [Kaiming He. 2015](https://arxiv.org/abs/1512.03385)
+Reference. [Learning Multiple Layers of Features from Tiny Images, Alex Krizhevsky, 2009](https://www.cs.toronto.edu/~kriz/learning-features-2009-TR.pdf)
+
+## How to run a train epoch
+
+### To simply run with a fake data.
+
+Once you compile, with `meson`, you can run with `meson test app_resnet18`.
+Please file an issue if you have a problem running the example.
+
+```bash
+$ meson ${build_dir} -Denable-test=true -Denable-long-test=true
+$ meson test app_resnet18 -v -c ${build_dir}
+```
+
```

Comment:
I guess it might be better to give more than just run. Maybe add some model summary and comparison result with TF in later PR.

Reply:
Sure, I'll add model description in later PR :)